src/imagery/i.eodag/i.eodag.html

```diff
@@ -0,0 +1,66 @@
+<h2>DESCRIPTION</h2>
+<H3>WARNING: I.EODAG IS STILL UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</H3>
+
+<em>i.eodag</em> allows to search and download imagery datasets, e.g. Sentinel, Landsat, and MODIS, from a number of different providers.
+The module utilizes the <a href="https://eodag.readthedocs.io/en/stable/">EODAG API</a>, as a single interface to search for datasets on the providers that EODAG supports.
+
+<p>
+To be able to download data through i.eodag, you will need to register for the providers you are interested in.
+See <a href="https://eodag.readthedocs.io/en/stable/getting_started_guide/register.html">providers registration</a>, and <a href="https://eodag.readthedocs.io/en/stable/getting_started_guide/configure.html">configure EODAG</a> for more details about configuring the providers' credentials.
+<p>
+
+<p>
+Currently, only products which footprint intersects current computation
+region extent (area of interest, AOI) are filtered.
+
+<p>
+To only list available scenes, <b>l</b> flag must be set. If no <b>start</b>
+or <b>end</b> dates are provided, the module will search scenes from the
+past 60 days.
+
+<p>
+To download all scenes found within the time frame provided, the user must
+remove the <b>l</b> flag and provide an <b>output</b> directory. Otherwise,
+files will be downloaded into <em>/tmp</em> directory.
+To download only selected scenes, one or more IDs must be provided
+through the <b>id</b> option file and an
+<b>output</b> directory.
+
+<h2>EXAMPLES</h2>
+
+Search available scenes:
+
+<div class="code"><pre>
+i.eodag  start=2022-05-25 end=2022-06-01 \
+    dataset=S2_MSI_L2A provider=cop_dataspace -l
+<pre></div>
+
+Download all available scenes:
+
+<div class="code"><pre>
+i.eodag  start=2022-05-25 end=2022-06-01 \
+    dataset=S2_MSI_L2A provider=cop_dataspace
+<pre></div>
+
+Download selected scenes by ID:
+
+<div class="code"><pre>
+i.eodag provider=cop_dataspace \
+    id=S2B_MSIL2A_20240526T080609_N0510_R078_T37SDD_20240526T094753,S2B_MSIL2A_20240529T081609_N0510_R121_T37SED_20240529T124818 \
+    -e config=custom/eodag_config.yaml
+<pre></div>
+
+<h2>REQUIREMENTS</h2>
+
+<ul>
+    <li><a href="https://eodag.readthedocs.io/en/stable/getting_started_guide/install.html">eodag library</a> (install with <code>pip install eodag</code>)</li>
```

Comment:
```suggestion
    <li><a href="https://eodag.readthedocs.io/en/stable/getting_started_guide/install.html">EODAG library</a> (install with <code>pip install eodag</code>)</li>
```

Reply:
Let's agree on how to write EODAG/eodag and keep it consistent through docs

Reply:
EODAG makes more sense to me.